src/langs/en.json

```diff
@@ -218,6 +219,7 @@
   "LIBRARY_SETTINGS_PUBLICATION_STATUS_INVALID": "Invalid",
   "LIBRARY_SETTINGS_PUBLICATION_STATUS_OUTDATED": "Outdated",
   "LIBRARY_SETTINGS_PUBLICATION_STATUS_UNPUBLISHED": "Unpublished",
+  "LIBRARY_SETTINGS_PUBLICATION_STATUS_TYPE_NOT_ALLOWED": "Item Type not allowed",
```

Comment:
I would have something more specific (file cannot be published, link cannot be published)

Reply:
Really good idea !